pkg/securitypolicy/regopolicy_test.go

```diff
@@ -2261,6 +2306,29 @@ type regoGetPropertiesTestConfig struct {
 	policy *regoEnforcer
 }
 
+func setupDumpStacksTest(constraints *generatedConstraints, allowDumpStacks bool) (tc *regoGetPropertiesTestConfig, err error) {
+	constraints.allowDumpStacks = allowDumpStacks
+
+	securityPolicy := constraints.toPolicy()
+	defaultMounts := generateMounts(testRand)
+	privilegedMounts := generateMounts(testRand)
+
+	policy, err := newRegoPolicy(securityPolicy.marshalRego(),
+		toOCIMounts(defaultMounts),
+		toOCIMounts(privilegedMounts))
+	if err != nil {
+		return nil, err
+	}
+
+	return &regoGetPropertiesTestConfig{
+		policy: policy,
+	}, nil
+}
+
+type regoDumpStacksTestConfig struct {
```

Comment:
we should have a generic one for other tests that require the minimal setup like here and in a few other tests. Lets add it to the nit list.

Reply:
Yes, this is on our list of things to look at when we do the big test refactor we discussed yesterday.